json/shared/src/main/scala/rapture/json/json.scala

```diff
@@ -40,7 +40,7 @@ private[json] class DynamicWorkaround(json: Json) {
 
 trait `Json.parse` extends MethodConstraint
 
-private[json] trait JsonDataCompanion[+Type <: JsonDataType[Type, AstType], AstType <: JsonAst]
+private[json] trait JsonDataCompanion[Type <: JsonDataType[Type, AstType], AstType <: JsonAst]
```

Comment:
Not sure about the change in variance, again. Might be worth having a chat about this...